clients/vscode-hlasmplugin/syntaxes/hlasmListing.tmLanguage.json

```
@@ -26,7 +26,7 @@
   ],
   "repository": {
     "listing_syntax":{
-      "match": "^(.{7})(.{9})(.{22})(\\d\\d|.\\d|.[+]|.{2})(.{71,72}).*$",
+      "match": "^(.{7})(.{9})(.{22})(\\d\\d|.\\d|.[+]|.{2})(.{1,72}).*$",
```

would the following work?
```suggestion
      "match": "^(.{7})(.{9})(.{22})(\\d\\d|.\\d|.[+]|.{2})(.{1,72})",
```